puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/gemset_create.yaml

```diff
@@ -0,0 +1,4 @@
+- builder:
+    name: gemset_create
+    builders:
+      - shell: !include-raw scripts/gemset_create.sh
```

Comment:
Test run where I removed all but the bundler install and set the rvm environment to ${ruby}@${EXECUTOR_NUMBER}

http://ci.theforeman.org/job/test_katello_core/database=postgresql,ruby=1.9.3,slave=fast/16669/console